app/ledger.js

```diff
@@ -981,8 +986,12 @@ var stickyP = (publisher) => {
   var result = (siteSetting) && (siteSetting.get('ledgerPayments'))
 
   // NB: legacy clean-up
-  if ((typeof result === 'undefined') && (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined')) {
-    result = synopsis.publishers[publisher].options.stickyP
+  if (typeof result === 'undefined') {
+    if (typeof synopsis.publishers[publisher].options.stickyP !== 'undefined') {
+      result = synopsis.publishers[publisher].options.stickyP
+    } else {
+      result = true
+    }
     appActions.changeSiteSetting(pattern, 'ledgerPayments', result)
   }
   delete synopsis.publishers[publisher].options.stickyP
```

Comment:
yep. It works until ledger recognizes it as a publisher. Once it does, publisher is set as enabled:

I did two checks on payments panel (shown in GIF below), on first try publisher wasn't there yet, so it was set as disabled. Once payments can _see_ the publisher, it gets enabled.

![auto_include_gif](https://cloud.githubusercontent.com/assets/4672033/23979178/aceb2fec-09d6-11e7-81bf-24b8034c96fe.gif)